app/models/conditions_response/backup.rb

```diff
@@ -30,7 +30,8 @@ class AbstractBackupMaker
   attr :backup_target_filebase, :backup_sources
 
   # Set the backup target and the backup sources
-  def initialize(backup_target_filebase: default_backup_filebase, backup_sources: default_sources)
+  def initialize(backup_target_filebase: default_backup_filebase,
```

Comment:
Shouldn't the 1st arg be `backup_target_filename` - that is actually what this arg represent (at least with this PR).  And, since this is used by `backup`, we want `backup` to received a filename to store the backup, and _not_ have to extend a firebase into an actual filename (that is, we're not doing that now, and we'd never want to do that in any case).

Reply:
this isn't being 'extended' in any way.  
But clearly my naming _is_ confusing. 
If no target filename is provided, then this will use the default for the Maker. 
Ex:  "current.tar" is the default target filename (base) for the Code Backup Maker

Let me revise and clarify.